bin/babel/index.js

```diff
@@ -29,6 +29,7 @@ commander.option("-R, --react-compat", "Makes the react transformer produce pre-
 commander.option("--keep-module-id-extensions", "Keep extensions when generating module ids", false);
 commander.option("-a, --auxiliary-comment [comment]", "Comment text to prepend to all auxiliary code");
 commander.option("-D, --copy-files", "When compiling a directory copy over non-compilable files");
+commander.option("--highlight [when]", "Use color in output (never, always or auto) [auto]", 'auto');
```

Comment:
Not sure about the argument name `highlight` since it's not very descriptive and the description doesn't give much either. It just controls color in syntax error stack traces.